File: abp/src/Wallee.Boc.Vote.Application.Contracts/Appraisements/AppraisementUpdateDto.cs
using System;
using Volo.Abp.Domain.Entities;

namespace Wallee.Boc.Vote.Appraisements
{
    public class AppraisementUpdateDto : IHasConcurrencyStamp
    {
        public string Name { get; set; } = null!;
        public string Description { get; set; } = null!;
        public AppraisementCategory Category { get; set; }
        public DateTime Start { get; set; }
        public DateTime End { get; set; }
        public string ConcurrencyStamp { get; set; } = null!;
    }
}

File: vote_vite_admin/src/openapi/models/AppraisementUpdateDto.ts
/* istanbul ignore file */
/* tslint:disable */
/* eslint-disable */

import type { AppraisementCategory } from './AppraisementCategory';

export type AppraisementUpdateDto = {
    name?: string | null;
    description?: string | null;
    category?: AppraisementCategory;
    start?: string;
    end?: string;
    concurrencyStamp?: string | null;
};
